dashboard/app/models/ability.rb

```diff
@@ -223,7 +237,9 @@ def initialize(user)
 
     can [:read, :student_lesson_plan], Lesson do |lesson|
       script = lesson.script
-      if script.pilot?
+      if script.in_development?
```

Comment:
This might've been called out in the spec but pdf generation won't work for these in development units/courses, right? Is this something we should warn curriculum writers about?

Also might be worth updating our PDF logic (in a separate PR) to filter for beta, preview, and stable units when deciding what to generate.

Reply:
Oh interesting point. I think that means it does not currently work for pilot courses and that in-development courses won't work either.  Added this to track that https://codedotorg.atlassian.net/browse/PLAT-1123

Reply:
I don't think we should block adding this because of the pdf generation but should look at fixing soon